spec/recursive_open_struct/ostruct_2_0_0_spec.rb

```diff
@@ -100,6 +100,46 @@
       end
     end
 
+    context "marshal support" do
```

Comment:
Why is this being tested in ostruct_2_0_0_spec.rb? I don't think marshalling support was added in Ruby 2.0. It looks like it existed in at least Ruby 1.9.3 as well.